src/blockdata/transaction.rs

```diff
@@ -673,7 +689,9 @@ impl SigHashType {
          }
      }
 
-     /// Reads a 4-byte uint32 as a sighash type
+     /// Reads a 4-byte uint32 as a sighash type.
+     /// Note: this replicates consensus behaviour, for current standardness rules correctness
```

Comment:
Idk if this is intentional, but the note will appear in the same line as the previous sentence. Maybe add a line break and make "Note:" bold for emphasis?